src/OpenApi/src/Extensions/JsonNodeSchemaExtensions.cs

```diff
@@ -86,7 +86,7 @@ internal static void ApplyValidationAttributes(this JsonNode schema, IEnumerable
         {
             if (attribute is Base64StringAttribute)
             {
-                schema[OpenApiSchemaKeywords.TypeKeyword] = "string";
+                schema[OpenApiSchemaKeywords.TypeKeyword] = JsonSchemaType.String.ToString();
```

Comment:
Is it right that this is changing from `string` to `String` (and for all these from lower to sentence case)? If it is, maybe using `nameof()` would be better to avoid the `ToString()` call at runtime (unless the compiler is smart enough to do that automatically?) when it's a constant value.

Reply:
> Is it right that this is changing from string to String (and for all these from lower to sentence case)? 

Depends on the definition of write. I primarily did this to make the values more resilient to being read by `Enum.Parse<JsonSchemaType>` without having case-insensitivity configured.

We _do_ set `ignoreCase: true` everywhere we parse these values and users should never interact with the opaque version of the schema but this feels...safer? 😓 

> Is it right that this is changing from string to String (and for all these from lower to sentence case)? If it is, maybe using nameof() would be better to avoid the ToString() call at runtime (unless the compiler is smart enough to do that automatically?) when it's a constant value.

The compiler doesn't map `ToString` to `nameof`. `nameof` would work for simple types but things might get funky when we support nullability via the `type` argument (`JsonSchemaType.String | JsonSchemaType.Null`) so `ToString` felt like the best approach to rely on consistently.